src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -131,4 +131,15 @@ public int hashCode() {
         return internalList.hashCode();
     }
 
+
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    String getPrintableString(Printable... printables) {
+        final String concatenatedPrintables = new String();
+        for (Printable p : printables) {
+            concatenatedPrintables.concat(p.getPrintableString());
```

Comment:
Can try out doing this using StringJoiner which makes your code "prettier" 
Read about it [here](https://stackoverflow.com/questions/27522563/why-stringjoiner-when-we-already-have-stringbuilder)